### PR TITLE
Feature/sum up walks

### PR DESCRIPTION
**Pull request by yannhn**

### CODE REVIEW

File: src/components/InputFood/index.js [hunk 1]
```diff
@@ -23,6 +23,9 @@ export default function InputFood({addNewFoodItem, setSubmittedFoodInput, setSub
 		resetForm();
 	};
 
+	const endHours = Number(enteredTime.split(':')[0]);
+	console.log(endHours);
```
Comment (stefankorder) [inline src/components/InputFood/index.js:27]: Please delete this console log. In your final PR should never be a console log.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! Take a look at my suggestions. 😃

**Comment by yannhn** [comment]
Thanks for the feedback @stefankorder ! I already implemented it. Somehow parts of my code landed in the wrong component (InputFood).

**Review by stefankorder — APPROVED** [review]
Good work!

**Pull request merged**